config-editor/config-editor-ui/src/app/components/config-manager/config-manager.component.ts

```diff
@@ -256,6 +284,36 @@ export class ConfigManagerComponent implements OnInit, OnDestroy {
         this.router.navigate([this.editorService.serviceName, 'edit'], { queryParams: { pasteConfig: true } });
       }
     });
+  }
+
+  getRowNodeId: GetRowNodeIdFunc = function (data) {
+    return data.config_name;
+  };
+
+  getRowData(configs$: Observable<Config[]>, release$: Observable<Release>): Observable<ConfigManagerRow[]> {
```

Comment:
move all logic code form components into store/services and make the component reactive